### PR TITLE
Added cluster utilization warning & critical alerts

### PR DESCRIPTION
**Pull request by shtripat**
ceph-mixins-bug-id: ceph/ceph-mixins/issues/3
ceph-mixins-bug-id: ceph/ceph-mixins/issues/4
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
@b-ranto @pcuzner please review

File: alerts/utilization.libsonnet [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'utilization-alert.rules',
+        rules: [
+          {
+            alert: 'CephClusterNearFull',
+            expr: |||
+              (ceph_osd_stat_bytes_used / ceph_osd_stat_bytes) * 100 > 85
```
Comment (b-ranto) [inline alerts/utilization.libsonnet]: afaik, you should use a `sum`, here, i.e.: `sum(ceph_osd_stat_bytes_used) / sum(ceph_osd_stat_bytes) > 0.85 `. (the same goes for the other rule)

**Review by b-ranto — CHANGES_REQUESTED** [review]

File: alerts/utilization.libsonnet (hunk 1 above)
Comment (shtripat) [reply alerts/utilization.libsonnet]: Ah yes. Will change the rules accordingly.

**Comment by shtripat** [comment]
@cloudbehl please review

**Comment by cloudbehl** [comment]
@shtripat Looks good to me but can we merge travis patch before merging this and make sure the manifest is up-to-date and things are not breaking.

**Comment by shtripat** [comment]
> @shtripat Looks good to me but can we merge travis patch before merging this and make sure the manifest is up-to-date and things are not breaking.

@cloudbehl As discussed, there are some other things we need to look at before merging the travis PRs for verification of rules file for PR, lets take this PR in and later we would try reviving the travis PR.

**Review by umangachapagain — APPROVED** [review]

**Pull request merged**